testData/solutions/win32.json5

```diff
@@ -5807,6 +6472,458 @@
+                "supportedSettings": {
+                    "ScrollWheelModeConfig": {
+                        "schema": {
+                            "title": "Mouse scroll wheel mode",
+                            "description": "Changes the number of lines to scroll when moving the scrollwheel.",
+                            "type": "integer",
+                            "default": 3,
+                            "minimum": 1,
+                            "oneOf": [
+                                {
+                                    "maximum": 100
+                                },
+                                {
+                                    enum: [4294967295],
+                                    enumLabels: ["full-page"]
+                                }
+                            ]
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "ScrollWheelModeConfig": {
+                        "transform": {
+                            "type": "fluid.transforms.value",
+                            "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.ScrollLines",
+                            "outputPath": "value"
+                        },
+                        "path": {
+                            "transform": {
+                                "type": "fluid.transforms.literalValue",
+                                "input": {
+                                    "get": "pvParam",
+                                    "set": "uiParam"
+                                }
+                            }
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
+                    "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.ScrollLines": "ScrollWheelModeConfig.value"
+                }
+            },
+            "configureScrollInactiveWindow": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETMOUSEWHEELROUTING",
+                    "setAction": "SPI_SETMOUSEWHEELROUTING",
+                    "uiParam": 0,
+                    "pvParam": {
+                        "type": "UINT"
+                    },
+                    "fWinIni": "SPIF_UPDATEINIFILE|SPIF_SENDCHANGE",
+                    "verifySettings": true
+                },
+                "supportedSettings": {
+                    "ScrollFocusRoutingConfig": {
+                        "schema": {
+                            "title": "Mouse scroll focus routing",
+                            "description": "Changes the way mouse scrolling is passed to applications",
+                            "type": "string",
+                            "enum": ["RoutingFocus", "RoutingHybrid"]
```

Comment:
This block all seems a bit elaborate - couldn't it be replaced with an enum with values [0, 1] and enumLabels of ["RoutingFocus", "RoutingHybrid"]?

Reply:
Yes, remembering our last talk with the @the-t-in-rtf the correct way to go with this should be simply to use the actual enum values, and give it the proper enumLabels. So we don't need to map the actual system values, and add no logic to the solution.